mapclientplugins/geometricfitstep/view/geometricfitwidget.py

```diff
@@ -514,48 +501,49 @@ def _updateGroupSettingWidgets(self):
-            self._updateDataProportion()
+            self._updateGroupConfigCentralProjection()
+            self._updateGroupConfigDataProportion()
         elif isFit:
-            self._updateDataWeight()
-            self._updateStrainPenalty()
-            self._updateCurvaturePenalty()
-        self._ui.configCentralProjection_checkBox.setVisible(isConfig)
-        self._ui.configSetCentralProjection_checkBox.setVisible(isConfig)
-        self._ui.configDataProportion_checkBox.setVisible(isConfig)
-        self._ui.configDataProportion_lineEdit.setVisible(isConfig)
-        self._ui.configDataWeight_checkBox.setVisible(isFit)
-        self._ui.configDataWeight_lineEdit.setVisible(isFit)
-        self._ui.configStrainPenalty_checkBox.setVisible(isFit)
-        self._ui.configStrainPenalty_lineEdit.setVisible(isFit)
-        self._ui.configCurvaturePenalty_checkBox.setVisible(isFit)
-        self._ui.configCurvaturePenalty_lineEdit.setVisible(isFit)
-
-    def _configSettingGroupChanged(self, index):
+            self._updateGroupFitDataWeight()
+            self._updateGroupFitStrainPenalty()
+            self._updateGroupFitCurvaturePenalty()
+        self._ui.groupConfigCentralProjection_checkBox.setVisible(isConfig)
+        self._ui.groupConfigSetCentralProjection_checkBox.setVisible(isConfig)
+        self._ui.groupConfigDataProportion_checkBox.setVisible(isConfig)
+        self._ui.groupConfigDataProportion_lineEdit.setVisible(isConfig)
+        self._ui.groupFitDataWeight_checkBox.setVisible(isFit)
+        self._ui.groupFitDataWeight_lineEdit.setVisible(isFit)
+        self._ui.groupFitStrainPenalty_checkBox.setVisible(isFit)
+        self._ui.groupFitStrainPenalty_lineEdit.setVisible(isFit)
+        self._ui.groupFitCurvaturePenalty_checkBox.setVisible(isFit)
+        self._ui.groupFitCurvaturePenalty_lineEdit.setVisible(isFit)
+
+    def _groupSettingChanged(self, index):
         """
         Callback for change in setting group field chooser widget.
         """
         self._updateGroupSettingWidgets()
 
-    def _getGroup(self):
-        group = self._ui.configSettingGroup_fieldChooser.getField()
+    def _getGroupSettingsGroupName(self):
+        group = self._ui.groupSettings_fieldChooser.getField()
+        groupName = None
         if group:
-            group = group.getName()
-        return group
+            groupName = group.getName()
+        return groupName
 
     def _getGroupSettingDisplayState(self, func):
-        group = self._getGroup()
+        group = self._getGroupSettingsGroupName()
         data, isLocallySet, inheritable = func(group)
         realFormat = "{:.4g}"
         lineEditDisable = True
         checkBoxTristate = False
         checkBoxState = QtCore.Qt.Unchecked
-        if isinstance(data, bool):
-            pass
+        if isinstance(data, float):
+            data = realFormat.format(data)
         elif isinstance(data, list):
             data = ','.join(str(e) for e in data)
```

Comment:
Shouldn't this use realFormat as well -- assuming real? Also, please change to ", ".join() i.e. space after comma for consistency with other lists in Align step.
```
", ".join(realFormat.format(e) for e in data)
```